locations/spiders/pizza_hut_us.py

```diff
@@ -50,7 +50,7 @@ def parse_store(self, response):
         ref = re.findall(r".com/(.+?)/(.+?)/(.+)", response.url)[0]
         ref = "_".join(ref)
         properties = {
-            "name": response.xpath('//span[@class="c-address-street-1"]/text()').extract_first(),
+            "branch": response.xpath('//span[@class="c-address-street-1"]/text()').extract_first(),
```

Comment:
```suggestion
```

Addresses aren't `branch`.

Reply:
well, in some of the previous PRs there was request to put such fake `branch` into `branch`

I would prefer to remove it altogether both here and there so I updated this PR.